workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -93,6 +97,7 @@ def __init__(
             value=None,
             description=self.metadata.label_description["undefined"],
             svg=self._get_default_figure(),
+            html=None,
```

Comment:
You have to make this variable optional or (preferably) use an empty string as initialization.